cmd/zitadel/startup.yaml

```diff
@@ -12,30 +12,30 @@ Log:
 Mgmt:
   API:
     GRPC:
-      ServerPort: 60020
-      GatewayPort: 60021
+      ServerPort: 50010
+      GatewayPort: 50011
       CustomHeaders:
         - x-caos-
 
 Auth:
   API:
     GRPC:
-      ServerPort: 60050
-      GatewayPort: 60051
+      ServerPort: 50020
+      GatewayPort: 50021
       CustomHeaders:
         - x-caos-
 
 Login:
-
+#  will get port range 5003x
 
 Admin:
   API:
     GRPC:
-      ServerPort: 60090
-      GatewayPort: 60091
+      ServerPort: 50040
+      GatewayPort: 50041
       CustomHeaders:
         - x-caos-
 
 Console:
-  Port: '9090'
-  StaticDir: '/app/console/dist'
+  Port: 50050
+  StaticDir: /app/console/dist
```

Comment:
Just a question. Would it not make sense to just expose GRPC, REST, HTML?
And deal the rest with the paths?

Reply:
as of now we create a listener (tcp port binding) for each server and gateway
we could try to use the same listener on the same service and even pass it to other services
the intention was to have flags controlling whether to start an api / ui

but from that perspective... it might be even better to change the package structure
and have:
```
pkg
    api
        mgmt
        auth
        admin
        login
        console
    api.go <- create listener / router and pass it to sub packages
    (more packages)
```

Reply:
Well my point us just that grpc service already discover there endpoints based on the svc name and rest does it by path as well.

Reply:
as discussed in the meeting today, i'll change this later as follows:
a single port for every of the three sections: grpc api, rest api, http (login and console)
we will handle this in a new PR